File: iPad/VerBank-CSTSv3-IOSClientApi/ClientStation/api/csts/APIReConnectInfoNode.m
//
//  APIReConnectInfoNode.m
//  JEDIv7-CSTSv3-ClientAPI
//
//  Created by baolin on 13/5/16.
//  Copyright (c) 2013年 wangyubo. All rights reserved.
//

#import "APIReConnectInfoNode.h"

@implementation APIReConnectInfoNode

-(NSString *)getIpAddress{
    return ipAddress;
}

-(void)setIpAddress:(NSString *)_ipAddress{
    ipAddress=_ipAddress;
}

-(int)getPort{
    return port;
}

-(void)setPort:(int)_port{
    port=_port;
}

-(long)getTime{
    return time;
}

-(void)setTime:(long)_time{
    time=_time;
}

-(Boolean)isSucceed{
    return succeed;
}

-(void)setSucceed:(Boolean)_isSucceed{
    succeed=_isSucceed;
}

-(int)getResult{
    return result;
}

-(void)setResult:(int)_result{
    result=_result;
}

-(long)getLoginUsedTime{
    return loginUsedTime;
}

-(void)setLoginUsedTime:(long)_loginUsedTime{
    loginUsedTime=_loginUsedTime;
}
@end

File: WeChatService/EX02-CSTSv3-ClientAPI/src/jedi/ex01/client/station/api/CSTS/APIReConnectInfoNode.java
package jedi.ex01.client.station.api.CSTS;

public class APIReConnectInfoNode {

	private long time;
	private String ipAddress;
	private int port;
	private boolean succeed;
	private int result;
	private long loginUsedTime;

	public long getTime() {
		return time;
	}

	public void setTime(long time) {
		this.time = time;
	}

	public String getIpAddress() {
		return ipAddress;
	}

	public void setIpAddress(String ipAddress) {
		this.ipAddress = ipAddress;
	}

	public int getPort() {
		return port;
	}

	public void setPort(int port) {
		this.port = port;
	}

	public boolean isSucceed() {
		return succeed;
	}

	public void setSucceed(boolean succeed) {
		this.succeed = succeed;
	}

	public int getResult() {
		return result;
	}

	public void setResult(int result) {
		this.result = result;
	}

	public long getLoginUsedTime() {
		return loginUsedTime;
	}

	public void setLoginUsedTime(long loginUsedTime) {
		this.loginUsedTime = loginUsedTime;
	}

}
